script/Ciel/Day 13/13_01_02_CIEL13_6_2.txt

```diff
@@ -242,6 +242,7 @@ Besides, I can't allow myself to die in a place like this.
 [sha:b3a513ffae31eb20e1dd87736ab6a93d9215b12c]{
 -- Page 199, Offset 35837.
 -- 俺は残り少ない血液と意識をかき集めて、覆い被さってくる死者の壁を切り裂いた。
+// Better alternative for consciousness here?
```

Comment:
buddy if i had that it would've been sprinkled all over day 5

Reply:
Maybe it can sound a bit more sane like this
>I muster every sliver of my consciousness and blood left in me...

Reply:
That sounds kinda spliced. I think you have to either leave it or more heavily split the two out, e.g. 
> I pull myself together, muster what blood I have left,
 and begin cutting...

Reply:
maybe removing "my" is good enough

>I muster every sliver of consciousness and blood left in me...